test/src/test/java/jenkins/model/JenkinsManagePermissionTest.java

```diff
@@ -0,0 +1,215 @@
+package jenkins.model;
+
+import java.net.HttpURLConnection;
+import com.gargoylesoftware.htmlunit.WebResponse;
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.MockAuthorizationStrategy;
+import org.jvnet.hudson.test.recipes.WithPlugin;
+
+import hudson.PluginWrapper;
+import hudson.cli.CLICommandInvoker;
+import hudson.cli.DisablePluginCommand;
+import hudson.model.Descriptor;
+import hudson.model.MyView;
+import hudson.model.View;
+import hudson.model.labels.LabelAtom;
+import hudson.tasks.Shell;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import static hudson.cli.CLICommandInvoker.Matcher.failedWith;
+
+/**
+ * As Jenkins.MANAGE can be enabled on startup with jenkins.security.ManagePermission property, we need a test class
+ * with this property activated.
+ */
+// TODO move tests to indicated test classes when we no longer need to set the system property
+public class JenkinsManagePermissionTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @BeforeClass
+    public static void enableManagePermission() {
+        System.setProperty("jenkins.security.ManagePermission", "true");
+    }
+
+    @AfterClass
+    public static void disableManagePermission() {
+        System.clearProperty("jenkins.security.ManagePermission");
+    }
+
+
+    // -----------------------------
+    // DisablePluginCommandTest
+    @Issue("JENKINS-60266")
+    @Test
+    @WithPlugin({ "depender-0.0.2.hpi", "dependee-0.0.2.hpi"})
+    public void managerCanNotDisablePlugin() {
```

Comment:
```suggestion
    public void managerCannotDisablePlugin() {
```

Reply:
is `cannot` a British Englishism?

Reply:
https://en.wiktionary.org/wiki/cannot#Usage_notes

Reply:
“[a] manager can not disable [a] plugin” reads to me something like “a manager is permitted to decline to disable a plugin”. “cannot” is what was meant here: “a manager is not permitted to disable a plugin”.